server/notification-providers/teams.js

```diff
@@ -1,5 +1,5 @@
 const NotificationProvider = require("./notification-provider");
-const axios = require("axios");
+const request = require("request");
```

Comment:
As @Zaid-maker said, it is deprecated.

Reply:
Yes it's deprecated and have high dependency vulnerabilities issues. So it's adviced to use axios in place